src/blockdev.rs

```diff
@@ -519,6 +520,32 @@ pub struct SavedPartitions {
     partitions: Vec<(u32, GPTPartitionEntry)>,
 }
 
+fn translate_mbr_types_to_gpt(sys: u8) -> Uuid {
+    // non inclusive best effort mapping of MBR types to GPT types
+    match sys {
```

Comment:
If you source this table from somewhere on the web, it would be good to put an URL in comments here.

Reply:
Thats a great point, I will add the wiki links in a comment, thank you :)